### ISSUE TITLE
Line height could be bigger and more attractive for readers

### ISSUE THREAD

**Issue by victorvalle** [liferay-design/liferay.design#905]
Everytime I read on Lexicon I find text really glued together, I believe it needs more air for better readability.

Current situation:

```
		p {
			line-height: 1.5;
			margin-bottom: $base-size * 0.5;

			a {
				color: $brand-primary;
			}
		}
```
Outcome:

![image](https://user-images.githubusercontent.com/21121115/106146905-8a0c5a80-6177-11eb-9984-9418ccdb9bad.png)

My proposal

```
		p {
			line-height: 1.75;
			margin-bottom: $base-size * 0.7;

			a {
				color: $brand-primary;
			}
		}

		ul,
		li {
			line-height: 1.5;
			margin-bottom: $base-size * 0.5;
		}
```

Outcome:
![image](https://user-images.githubusercontent.com/21121115/106146558-19fdd480-6177-11eb-8e07-d7384b36434b.png)

cc: @drakonux @hold-shift

**Comment by hold-shift** [comment]
Right, I think we should work on the general visual design (font styling & images) to help users digest the content. That's why I proposed to work all the different sections of the site on the Figma document you have shared. 

I truly think this is the best way to see the website's visual aspect in the same way that we do with the product that we design. By skipping stages, we do nothing but penalize the result.

Having all the contents & assets located in the same place will also help us have a better vision of what everything looks like before we publish, and it would improve the content review workflow.

**Comment by victorvalle** [comment]
Probably this is a better example. The current version is on the right side

![image](https://user-images.githubusercontent.com/21121115/106158608-36543e00-6184-11eb-8655-9cf6fd2a20eb.png)

**Comment by victorvalle** [comment]
I see your point @hold-shift

**Comment by drakonux** [comment]
I think the line-height is one of the things we would change for sure. I agree the new version looks better, my concern is the impact of changing that how it's gonna affect (but if you have tested in local, go ahead). 

I agree with Ruben all in one place would give us a better understanding of the final result. We should create the epic for redesigning the site soon and create tasks to eat the elephant.

**Comment by plhnk** [comment]
Just a note — we currently have styles defined in a few different places:

1. `theme/lexicon.module.scss` ==> lexicon specific styles
2. `theme/documentation.module.scss` ==> general styling for the documentation sites (blueprints, handbook, lexicon 1 & 2)
3. `theme/global.scss` ==> global...styles..which probably affect things
4. `theme/index.js` ==> our first js theme, lots of redundancy w/the scss files and theme-ui
5. `theme/variables.scss` ==> variables for colors, spacing, etc that are referenced in a lot of places
6. `gatsby-plugin-theme-ui` ==> new theme stuff lives here

basically — we have been slowly moving things out of `#1-5` and into `#6` — but should use a new site design effort to completely clean/migrate everything into a more maintainable structure.

**Comment by victorvalle** [comment]
https://github.com/liferay-design/liferay.design/pull/907/commits/1d67de4443293c2fa6b601f121480ef92c3e5f90

**Comment by plhnk** [comment]
closing this issue as @victorvalle added a generous amount of line-height in #907
